guides/common/modules/proc_creating-a-resource-quota.adoc

```diff
@@ -0,0 +1,11 @@
+[id="creating-a-resource-quota"]
+= Creating a resource quota
+
+Create a resource quota to limit the resource consumption of a {Project} user.
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Configure* > *Resource Quotas*.
+. Click *Create resource quota*.
++
+Follow the wizard in the {ProjectWebUI}.
```

Comment:
I understand that you want to keep it compact and not to document a well-designed UI. At the same time, I feel strongly that "Follow the wizard" misses the mark. Instead of that phrase, I would write this:

Specify the resource quota name, CPU cores, and memory, and then click **Create resource quota**.

I prefer this approach because it provides all the information while only requiring a single step. 

I don't think you need to mention mandatory fields or how to use the toggle buttons. Most users are aware of these UI elements. The help text presumably explains the fields. Would be nice if the UI had "Quota limit" instead of "Quota Limit" for consistency but that's another issue.